src/Symfony/Component/Ldap/Tests/Adapter/ExtLdap/AdapterTest.php

```diff
@@ -178,14 +178,14 @@ public function testLdapPagination()
             $paged_results = $paged_query->execute();
 
             // All four of the above queries should result in the 25 'users' being returned
-            $this->assertEquals($unpaged_results->count(), 25);
-            $this->assertEquals($fully_paged_results->count(), 25);
-            $this->assertEquals($paged_results->count(), 25);
+            $this->assertCount(25, $unpaged_results);
```

Comment:
Can we use camelCase here for the variable names please? Thanks

Reply:
Renamed and inlined some vars 👍